_config.yml

```diff
@@ -33,8 +33,8 @@ description: "Software Carpentry is volunteer non-profit organization dedicated
 # See > authors.yml
 author: gvwilson
 
-# This URL identifies the domain.
-url: 'http://software-carpentry.org'
+# This URL identifies the domain.  It uses a protocol-independent path so that things will load with both HTTP and HTTPS.
+url: '//software-carpentry.org'
```

Comment:
On Wed, Dec 30, 2015 at 01:47:25PM -0800, Raniere Silva wrote:
> > +# This URL identifies the domain.  It uses a protocol-independent path so that things will load with both HTTP and HTTPS.
> > +url: '//software-carpentry.org'
> 
> Maybe forcing HTTPS?

This will work too for software-carpentry.org, although that will
*really* break local dev testing ;).